docs/webiny-apps/page-builder/development/creating-iframe-element-plugin.md

```diff
@@ -16,7 +16,7 @@ Ready? Let's dive in!
 
 We’ll build an element plugin that will allow content creators to embed content using an iframe.
 
-Here's what the result will look like
+Here's what the result will look like:
 
 ![Embed content with an iframe](/img/webiny-apps/page-builder/development/development/plugin-reference/editor/iframe/iframe-plugin.png)
```

Comment:
iFrame element/plugin? It just represent the name of the element we will create. What do you think, should we change that?

Reply:
It's kinda confusing to me, I don't know what this represents, is it related to the end-goal / the page element we're about to create, or is this some random page content that the user might've created?

If I understood this correctly, the end page element that will be rendered on the page is the map. 

If you wanted to just added some content on the page, just to give some context, I would rather have some text / sections that doesn't mention some technical terms, but mentions something we might see in a real life. For example, instead of the `iFrame plugin`, I would rather have "Where can you find us?", and then we have the iframe map, which is what we've created as part of this tutorial.

Again, I might have misunderstood something here, so feel free to correct me :P

Reply:
Yes it was just to give some content on the page, I can change it to the title you provided.

Reply:
I deleted the text click here, and the icon, I just used the title, do you think it's better to include more elements?

Reply:
It's okay, let's see how it'll look and if it's not too empty.